README.md

```diff
@@ -178,41 +178,50 @@ The only change on calculate routes is the `selects` paramater contains the func
 
 # Testing
```

Comment:
So this is an example for the user to test their StandardAPI integration. Perhaps we should have to sections, on for testing your application and the other titled testing standardAPI?